doc/connection-to-the-internet-of-things/how-to-read-measures-captured-from-iot-devices.md

```diff
@@ -1,121 +1,241 @@
+      'X-Auth-Token' :      '[TOKEN]',
+      'Fiware-Service':     'openiot',
+      'Fiware-ServicePath': '/'
+    }
+    
     Payload:
+    
     {
       "services": [
         {
-        "apikey": "4jggokgpepnvsb2uv4s40d59ov",
-        "token": "token2",
-        "cbroker": "http://0.0.0.0:1026",
-        "entity_type": "thing",
-        "resource": "/iot/d"
+          "apikey":      "4jggokgpepnvsb2uv4s40d59ov",
+          "token":       "aToken",
+          "cbroker":     "http://0.0.0.0:1026",
+          "entity_type": "thing",
+          "resource":    "/iot/d"
         }
       ]
     }
 
-Where 0.0.0.0:1026 might be replaced by a private instance of
-ContextBroker or just leave it like this (0.0.0.0:1026) if the public
-instance at the same VM (130.206.80.40:1026) will be used. The apikey
-string should be updated with a shared secret to be known by devices.
+Where `0.0.0.0:1026` might be replaced by a private instance of
+a Context Broker or just leave it as it is (`0.0.0.0:1026`) if the public
+instance is running at the same VM (`130.206.80.40:1026`) will be used. The apikey
+string should be updated with a shared secret to be known by your IoT devices. An OAuth
+token is needed as well, a simple way of obtaining one
+is described [here](http://fiware-orion.readthedocs.io/en/develop/quick_start_guide/index.html).
+
+Likewise, you may want to experiment using the *FIWARE Tour Guide Application*.
+Assuming you have installed it locally, you can follow these steps: 
+
+  POST http://localhost:4041/iot/services/
+  
+  Headers:
+      
+      {
+        'Content-Type':       'application/json’,
+        'Fiware-Service':     'tourguide',
+        'Fiware-ServicePath': '/'
+      }
+      
+      Payload:
+      
+      {
+        "services": [
+          {
+            "apikey":   "tourguide-devices",
+            "cbroker":  "http://orion:1026",
+            "resource": "/iot/dev-restaurants"
+          }
+        ]
+      }
+
+
+2. **Register your IoT device**
+
+ Before your device sends observations or receives commands a register operation is needed:
+
+    POST http://130.206.80.40:5371/iot/devices
+    
+    Headers:
+    
+    {
+      'Content-Type':       'application/json’,
+      'X-Auth-Token' :      '[TOKEN]',
+      'Fiware-Service' :    'openiot',
+      'Fiware-ServicePath': '/'
+    }
+    
+    Payload:
+    
+    {
+      "devices": [
+        {
+          "device_id":   "[DEV_ID]",
+          "entity_name": "[ENTITY_ID]",
+          "entity_type": "thing",
+          "timezone":    "Europe/Madrid",
+          "attributes": [
+            {
+              "object_id": "t",
+              "name":      "temperature",
+              "type":      "number"
+            }
+          ],
+          "static_attributes": [
+            {
+              "name":  "attr_name",
+              "type":  "string",
+              "value": "value"
+            }
+          ]
+        }
+      ]
+    }
+
```

Comment:
yes, you are right, probably we are going to need to align both

what about doing it in the context of https://github.com/Fiware/docs.TourGuide/pull/24 and we fix that issue following a similar approach?

I think we could do that at least for POST requests and for GET requests just mention it